src/app/banking/vouchers/search-redeemed-vouchers.component.ts

```diff
@@ -33,13 +33,15 @@ export class SearchRedeemedVouchersComponent
   }
 
   protected getFormControlNames(): string[] {
-    return ['types', 'periodBegin', 'periodEnd'];
+    return ['types', 'periodBegin', 'periodEnd', 'operator'];
   }
 
   protected toSearchParams(value: any): UserVoucherSearchParams {
     const params = cloneDeep(value);
```

Comment:
According to the last discussion we had, we would no longer clone the value on toSearchParams